### PR TITLE
Nav links

### PR DESCRIPTION
**Pull request by YinnyF**
[S] NavBar Links: Fix Routes - remove hardcode for localhost

### CODE REVIEW

File: app/views/layouts/_nav_bar.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <span aria-hidden="true"></span>
+    <span aria-hidden="true"></span>
+    </a>
+</div>
+<div class="navbar-menu" id='nav-links'>
+    <div class="navbar-start">
+    <a class="navbar-item" href="/posts"> Home </a>
+    <a class="navbar-item" href="/pages/about"> About </a>
+        <%# Drop down menu for nav bar %>
+        <%# <div class="navbar-item has-dropdown is-hoverable">
```
Comment (1ugia) [inline app/views/layouts/_nav_bar.html.erb]: Would we like to keep these commented out section?

**Review by 1ugia — CHANGES_REQUESTED** [review]
Great work :D

File: app/views/layouts/_nav_bar.html.erb (hunk 1 above)
Comment (YinnyF) [reply app/views/layouts/_nav_bar.html.erb]: You're right :D Thanks Mas, all updated now.

**Review by 1ugia — APPROVED** [review]
Laaavlyyyy :D well done guys!

**Pull request merged**